app/ui/archiveTile.js

```diff
@@ -291,9 +291,11 @@ module.exports.wip = function(state, emit) {
             ${state.translate('addFilesButton')}
           </label>
           <div class="font-normal text-sm text-grey-darker">
-            ${state.translate('totalSize', {
-              size: bytes(state.archive.size)
-            })}
+            ${platform() === 'android'
```

Comment:
the string causes a linebreak on android views...i think we can leave it out there